### PR TITLE
71 render qa list

### PR DESCRIPTION
**Pull request by amberlyM**
-rendering question list
-rendering answers per question 
-utilizing example data for rendering

Trello: https://trello.com/c/GT8StuSs/71-qa-on-page-load-up-to-four-questions-should-be-displayed-m

### CODE REVIEW

File: client/src/components/QA/QA.jsx [hunk 1]
```diff
@@ -5,15 +5,70 @@ import QuestionList from './Questions/QuestionList.jsx';
 class QA extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
```
Comment (hepnerthomas) [inline client/src/components/QA/QA.jsx:8]: looks good - though I think you'll probably delete this when you pull from the API

File: client/src/components/QA/QA.jsx [hunk 2]
```diff
@@ -5,15 +5,70 @@ import QuestionList from './Questions/QuestionList.jsx';
...
+            },
+          },
+        },
+      ],
+    };
   }
 
   render() {
+    let display;
+    const { results } = this.state;
```
Comment (hepnerthomas) [inline client/src/components/QA/QA.jsx:60]: `questions` would be more descriptive name than `results`

File: client/src/components/QA/Questions/Answers/AnswerList.jsx [hunk 3]
```diff
@@ -5,14 +5,40 @@ import AListCSS from './AnswerList.module.css';
 class AnswerList extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      results: [
```
Comment (hepnerthomas) [inline client/src/components/QA/Questions/Answers/AnswerList.jsx:9]: `answers` would be more descriptive than `results`

**Review by hepnerthomas — APPROVED** [review]
Nice work, @amberlyM ! Just left some style comments for improvement, though happy to merge upon request.

File: client/src/components/QA/QA.jsx (hunk 1 above)
Comment (amberlyM) [reply client/src/components/QA/QA.jsx:8]: Yes of course! its all for rendering right now

File: client/src/components/QA/QA.jsx (hunk 2 above)
Comment (amberlyM) [reply client/src/components/QA/QA.jsx:60]: 'result' comes straight from the data, wanted it to match for now but might change it later if I don't need to send and POST request from here.

File: client/src/components/QA/Questions/Answers/AnswerList.jsx (hunk 3 above)
Comment (amberlyM) [reply client/src/components/QA/Questions/Answers/AnswerList.jsx:9]: I wanted the data to match precisely for now, makes it easier to send data later if needed. But will change later if it becomes redundant. Thanks for feedback! Most of these things will be changed later. But would you merge when you get a min?

**Pull request merged**